src/transactions/test/InflationTests.cpp

```diff
@@ -283,12 +283,18 @@ doInflation(Application& app, int ledgerVersion, int nbAccounts,
     REQUIRE(expectedWinnerCount == payouts.size());
 }
 
-TEST_CASE("inflation", "[tx][inflation]")
+TEST_CASE("inflation total coins", "[tx][inflation]")
 {
     Config cfg = getTestConfig(0);
+    if (protocolVersionIsBefore(cfg.TESTING_UPGRADE_LEDGER_PROTOCOL_VERSION,
```

Comment:
Why is this needed? As far as I can tell, this test works with this removed.

Reply:
You're right that it's not needed. I added it because the test does nothing post-V11. We probably have a ton of tests where something like this would help, so I could remove this in favor of a more general solution in the future.